src/main/scala/com/linkedin/feathr/offline/util/SourceUtils.scala

```diff
@@ -234,51 +234,6 @@ private[offline] object SourceUtils {
     field.defaultVal()
   }
 
-  /* Defines a symmetric relationship for two keys regarding to that target fields, for example
-   * ( (viewerId, vieweeId), affinity ) <=> ( (vieweedId, viewerId), affinity ), so in the dataset,
-   * they are only stored once on HDFS, here this operation should generate the full data
-   */
-  def getRDDViewSymmKeys(rawRDD: RDD[_], targetFields: Option[Seq[String]], otherFields: Option[Seq[String]] = None): RDD[_] = {
-
```

Comment:
This was unneeded?

Reply:
It was never used, hence cleaning it up.